### DISCUSSION TITLE
AOD Tweaks

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [MonwF/customiuizer#38] [Ideas]
If you notice the Miui AOD app needs some additional features.
Like the AOD only worka for 10seconds and I've made a tweak to put a custom value for the timing like 1000seconds or so. If you want I can share the code and also some features are required in it.
As you can see like this one
https://forum.xda-developers.com/t/xposed-11-apr-v3-3-opaodmod-always-on-display-with-lots-of-options.4100981/

**Comment by MonwF** [comment]
no plan. you may keep it an independent module

**Reply by RevealedSoulEven** [reply]
What about adding it now?

**Reply by MonwF** [reply]
the same answer
